firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -252,6 +246,17 @@ static percent_t currentEtbDuty;
 		percent_t etbIdleAddition = CONFIGB(useETBforIdleControl) ? engine->engineState.idle.etbIdleAddition : 0;
 		percent_t targetPosition = engine->engineState.targetFromTable + etbIdleAddition;
 
+
+//This prevents ETB to hit hard-stops
+
+		if (targetPosition > 99.0f) {
```

Comment:
this is not related to deviation error-handling, I am mostly afraid of 1% since it's quite a large number

Reply:
The 99% are most important, as the idle-addition on top of target position brings you over 100%

The 1 % have not made any issues for me, but I know TPS calibration varies a lot due to mechanical play. At the ETB I have here now, it varies up to 2%.

So staying away from the hard-stop area might be a good idea.

We can adjust it down if you think it's high.

Reply:
1% is pretty small - though we could make it 0.5% or something if we're worried.

Reply:
I think we also have a `clamp()` function or something that clamps a value between two limits.  Clear intent is important, and a function called `clamp` explains that well.

Reply:
I dont think we should ever operate within the 1% area, but we can change to 0.5% to be sure if you want.

Reply:
The high side stop can probably actually come down to 95% or something - some throttles actually even go slightly past fully parallel for exactly that reason.